.storybook/config.js

```diff
@@ -3,9 +3,9 @@ import { configure } from "@storybook/react";
 // automatically import all files ending in *.stories.js
 // https://webpack.js.org/guides/dependency-management/
 const components = require.context("../packages/components/", true, /.stories.js$/);
-// const demo = require.context("../demo/", true, /.stories.js$/);
+const demo = require.context("../demo/", true, /.stories.js$/);
```

Comment:
lint these btw - single quotes etc.

Reply:
ok, cool changed it,

I think those storybookbook config are missed out by the linting 🤷‍♂️